typescript/README.md

```diff
@@ -1 +1,178 @@
 # TypeScript Best Practices
+
+1. **Consistent Coding Style**:
```

Comment:
I think that the official lint recommended configuration is : 
https://github.com/typescript-eslint/typescript-eslint
https://typescript-eslint.io/getting-started
https://typescript-eslint.io/users/configs/

I think we dont need again to use "airbnb-typescript".